web-client/src/pages/adding-work.js

```diff
@@ -0,0 +1,476 @@
+                <MenuItem className={style.MenuItem} value="">
+                  <em>Select</em>
+                </MenuItem>
+                <MenuItem className={style.MenuItem} value={"Computer Science"}>
+                  Computer Science
+                </MenuItem>
+                <MenuItem className={style.MenuItem} value={"Mathematics"}>
+                  Mathematics
+                </MenuItem>
+                <MenuItem className={style.MenuItem} value={"Biology"}>
+                  Biology
+                </MenuItem>
+                <MenuItem className={style.MenuItem} value={"Chemistry"}>
+                  Chemistry
+                </MenuItem>
+                <MenuItem className={style.MenuItem} value={"Psychics"}>
+                  Psychics
+                </MenuItem>
+                <MenuItem className={style.MenuItem} value={"Geography"}>
+                  Geography
+                </MenuItem>
+              </Select>
+              {hasError && <FormHelperText>Required field</FormHelperText>}
+            </FormControl>
+
+            {/* Authors */}
+            <TextField
+              className={style.textFieldAuthor}
+              inputRef={register}
+              id="authors-adding-work"
+              name="authors"
+              label="Authors"
+              autoComplete="authors"
+              InputLabelProps={{
+                shrink: true,
+              }}
+              value={mainAuthor}
+              variant="outlined"
+            />
+
+            {authorList}
+
+            {/* Acceptance - Rules of Conference */}
+            <FormControlLabel
+              className={style.formControlLabel}
+              control={
+                <Checkbox
+                  inputRef={register}
+                  required
+                  id="acceptance-signup"
+                  name="acceptance"
+                  color="primary"
+                />
+              }
+              label="I accept the Rules of Scienture Conference and I agree to processing my personal data included in the above form by...*"
+              inputRef={register}
+              name="acceptance"
+            />
+            <FormHelperText error className={style.formHelperText}>
+              {errors.acceptance ? errors.acceptance.message : " "}
+            </FormHelperText>
+
+            <Button
+              className={style.addButton}
+              color="primary"
+              type="submit"
+              variant="contained"
+              onClick={ClickSubmit}
+            >
+              Add work
+            </Button>
+          </div>
+          <div className={style.right}>
+            <DropZone SetFile={passFile} />
+          </div>
+        </form>
+      </div>
+      <Snackbar
+        open={openAlertError}
+        autoHideDuration={duration}
+        onClose={CloseAlert}
+      >
+        <Alert onClose={CloseAlert} severity={"error"}>
+          {"You did not choose a work!"}
+        </Alert>
```

Comment:
unnecessary tab

Reply:
This is necessary tab between Alert tags. Even code formatting forces this tab.